group_vars/location_fffw_lebenshilfe/networks.yml

```diff
@@ -0,0 +1,82 @@
+---
+#ipv6_prefix: "2001:bf7:860::/56" !anpassen!!
+
+networks:
+
+  - vid: 10
+    role: mesh
+    name: mesh_nno
+    prefix: 10.30.96.43/32
+    #ipv6_subprefix: -1
+    mesh_metric: 1024
+    mesh_ap: fffw-lebenshilfe-nno-ap-2ghz
+    mesh_radio: 11g_standard
+    mesh_iface: mesh
+
+  - vid: 11
+    role: mesh
+    name: mesh_nw
+    prefix: 10.30.96.44/32
+    #ipv6_subprefix: -2
+    mesh_metric: 1024
+    mesh_ap: fffw-lebenshilfe-nw-ap-2ghz
+    mesh_radio: 11g_standard
+    mesh_iface: mesh
+
+  - vid: 12
+    role: mesh
+    name: mesh_sso
+    prefix: 10.30.96.45/32
+    #ipv6_subprefix: -3
+    mesh_metric: 1024
+    mesh_ap: fffw-lebenshilfe-sso-ap-2ghz
+    mesh_radio: 11g_standard
+    mesh_iface: mesh
+
+  - vid: 13
+    role: mesh
+    name: mesh_ono
+    prefix: 10.30.96.46/32
+    #ipv6_subprefix: -4
+    mesh_metric: 1024
+    mesh_ap: fffw-lebenshilfe-ono-ap-2ghz
+    mesh_radio: 11g_standard
+    mesh_iface: mesh
+  
+  - vid: 14
+    role: mesh
+    name: mesh_uplink
+    prefix: 10.30.96.47/32
+    #ipv6_subprefix: -4
+    mesh_metric: 1024
+    mesh_iface: mesh
+
+  - vid: 42
+    role: mgmt
+    prefix: 10.30.96.32/28
+    gateway: 1
+    dns: 1
+    #ipv6_subprefix: 1
+    assignments:
+      fffw-lebenshilfe-core: 1
+      #fffw-lebenshilfe-switch: 2
+      fffw-lebenshilfe-nw-ap-2ghz: 3
+      fffw-lebenshilfe-nno-ap-2ghz: 4
+      fffw-lebenshilfe-sso-ap-2ghz: 5
+      fffw-lebenshilfe-ono-ap-2ghz: 6
+
+  - vid: 40
+    role: dhcp
+    prefix: 10.30.99.0/25
+    #ipv6_subprefix: 0
+    assignments:
+      fffw-lebenshilfe-core: 1
+
+location__channel_assignments_11g_standard__to_merge:
+  # ap-name: Channel  Bandwidth  tx-Power
+  fffw-lebenshilfe-nno-ap-2ghz: 1-20-15
```

Comment:
is there a reason why you set txpower explicit?

Reply:
No, there isn't. i just copied that from the documentation, which was derived from kiehlufer-core, I think... Can I omit the tx-power and it gets set automatically then?

Reply:
tx-power removed in new version.